core-common/src/main/java/org/apache/kylin/common/persistence/ResourceStore.java

```diff
@@ -474,7 +474,7 @@ public Object call() throws IOException {
      * delete a resource, does nothing on a folder
      */
     final public void deleteResource(String resPath) throws IOException {
-        logger.trace("Deleting resource {} (Store {})", resPath, kylinConfig.getMetadataUrl());
+        logger.info("Deleting resource " + resPath + " (Store " + kylinConfig.getMetadataUrl() + ")");
```

Comment:
I doubt there will be too much log of this one

Reply:
OK. Revert it. Thanks~